src/VkBootstrap.cpp

```diff
@@ -610,7 +610,7 @@ detail::Result<Instance> InstanceBuilder::build() const {
 	bool supports_properties2_ext =
 	    detail::check_extension_supported(system.available_extensions, VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME);
 
-	if (supports_properties2_ext) {
+	if (supports_properties2_ext && api_version == VK_API_VERSION_1_0) {
```

Comment:
Should be `api_version < VKB_VK_API_VERSION_1_1`
This way anyone setting a patch version isn't screwed and it uses the macro defines which correctly use the right version macro.